.github/renovate.json5

```diff
@@ -0,0 +1,28 @@
+{
+	"extends": [
+		"config:base",
+		"schedule:weekly", // each monday
+		"group:allNonMajor", // batch non-majors into one PR
+		":semanticCommits" //nicer commit messages
+	],
+	"labels": ["dependencies"],
+	"ignorePaths": [
+		"packages/playground/**",
+		"**/__tests__/**"
+	],
+	"pin": false,
+	"rangeStrategy": "bump",
+	"node": false,
+	"packageRules": [
+		{
+			"depTypeList": ["peerDependencies"],
```

Comment:
pngwn brought up about this yesterday, `depTypeList` doesn't seem to be [documented](https://docs.renovatebot.com/configuration-options) (likely deprecated API), maybe we want to change it to [matchDepTypes](https://docs.renovatebot.com/configuration-options/#matchdeptypes)

Reply:
thanks for pointing that out. updated to matchDepTypes